spec/src/main/asciidoc/javase/bootstrap_se.asciidoc

```diff
@@ -1,61 +1,133 @@
+    public SeContainerInitializer addPackages(boolean scanRecursively, Package... packages);
+    public SeContainerInitializer addAnnotatedTypes(AnnotatedType<?>... annotatedTypes);
+    public SeContainerInitializer addExtensions(Extension... extensions);
+    public SeContainerInitializer addExtensions(Class<? extends Extension>... extensions);
+    public SeContainerInitializer addInterceptors(Class<?>... interceptorClasses);
+    public SeContainerInitializer addDecorators(Class<?>... decoratorClasses);
+    public SeContainerInitializer addAlternatives(Class<?>... alternativeClasses);
+    public SeContainerInitializer addAlternativeStereotypes(Class<? extends Annotation>... alternativeStereotypeClasses);
+    public SeContainerInitializer addProperty(String key, Object value);
+    public SeContainerInitializer setProperties(Map<String, Object> properties);
+    public SeContainerInitializer addBeans(Bean<?>... beans);
+    public SeContainerInitializer disableDiscovery();
+    public SeContainerInitializer setClassLoader(ClassLoader classLoader);
+    public SeContainer initialize();
+}
+----
+
+Unless specified differently each methods of `SeContainerInitializer` returns the current `SeContainerInitializer` object.
+
+* `newInstance()` static method returns an instance of the implementation of `SeContainerInitializer` discovered by java service-provider.
+Each call returns a new instance of `SeContainerInitializer`.
+* `addBeanClasses()` add classes to the the synthetic bean archive
+* `addPackages()` add packages content to the synthetic bean archive.
+Versions exist by providing classes in the wished package or `Package` objects themselves.
+Other version of the method allows recursive addition
+* `addAnnotatedTypes()` add the provided `AnnotatedTypes` to the synthetic bean archive.
+* `addExtensions()` add the provided extensions (class or instance) to the synthetic bean archive.
+* `addInterceptors()` add the provided interceptors to the synthetic bean archive.
+* `addDecorators()` add the provided decorators to the synthetic bean archive.
+* `addAlternatives()` add the provided alternatives classes to the synthetic bean archive.
+* `addAlternativeStereotypes()` add the provided stereotypes classes to the synthetic bean archive.
+* `addProperty()` add a configuration property for the container
+* `SetProperties()` set the Map of configuration properties for the container.
+replace previous properties.
+* `addBeans()` add the provided beans to the set of discovered beans.
+* `disableDiscovery()` deactivate automatic type scanning and discovery.
+All bean archive will be ignored except the implicit bean archive.
+* `setClassLoader()` change the default class loader for the container
+* `initialize()` bootstrap the container and returns a `SeContainer` as defined in <<se_container>>.
+
+
+
+Each time `initialize()` method is called on the `SeContainerInitializer` it returns a new `SeContainer<Object>` instance.
```

Comment:
Every invocation of the `SeContainerInitializer.initialize()` method returns a new `SeContainer` instance.

Note that `SeContainer` is not parameterized.